cockatrice/src/dlg_add_set.cpp

```diff
@@ -0,0 +1,37 @@
+#include "dlg_add_set.h"
+
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+DlgAddSet::DlgAddSet(QWidget *parent, bool success) : QDialog(parent) {
+    status = new QLabel(this);
+    restart = new QLabel(this);
+
+    if (success) {
```

Comment:
I'm not sure it makes sense to check this on instantiation. It seems like that should be a separate modal.

A workflow like

Settings panel > click add custom set button > open qt file picker > dialog for result makes the most sense.

Maybe I'm getting confused from the class name? Is this actually an AddCustomSetResult?